kpi/deployment_backends/mock_backend.py

```diff
@@ -105,7 +105,7 @@ def generate_uuid_for_form():
     def form_uuid(self):
         return 'formhub-uuid'  # to match existing tests
 
-    def nlp_tracking_data(self, start_date=None):
+    def nlp_tracking_data(asset_ids=None):
```

Comment:
is this a mistake? i see `start_date` still referenced in the method (and `asset_ids` does not appear)

Reply:
It is indeed. It should be 

```python 
@staticmethod
def nlp_tracking_data(asset_ids=None, start_date=None):
    ...
```

but this method is gone in #5056. Fixing it here will cause conflicts when this branch is merged in #5056. Is it really worth it? 


Out of scope: It really means the tests were not covering this part.

Reply:
if it's gone in #5056, let's not worry about it :)